conda_env/cli/main_export.py

```diff
@@ -45,7 +45,10 @@ def configure_parser(sub_parsers):
     p.add_argument(
         '-f', '--file',
         default=None,
-        required=False
+        required=False,
+        help="File name or path for the exported environment. "
+        + "Note that if you want to export your environment to a file named `environment.yml`, "
+        + "an already existing `environment.yml` will be overwritten.",
```

Comment:
I'm in favor of being more concise here and using implicit string concatenation over `+`:

```suggestion
        help=(
            "File name or path for the exported environment. "
            "Note: this will silently overwrite any existing file."
        ),
```

Reply:
That works for me! I'd capitalize "This" after the colon as well.